frontend/packages/kubevirt-plugin/src/components/form/size-unit-utils.tsx

```diff
@@ -0,0 +1,11 @@
+import { assureEndsWith } from '@console/shared/src';
+
+export enum BinaryUnit {
```

Comment:
We already have a units file, can we use the public component's units here ?
https://github.com/openshift/console/blob/master/frontend/public/components/utils/units.js#L22

Reply:
These ones are typed and you do not need an index to reference them